sys/net/network_layer/ng_sixlowpan/iphc/ng_sixlowpan_iphc.c

```diff
@@ -499,7 +499,7 @@ bool ng_sixlowpan_iphc_encode(ng_pktsnip_t *pkt)
         }
 
         if ((src_ctx != NULL) || ng_ipv6_addr_is_link_local(&(ipv6_hdr->src))) {
-            eui64_t iid = { 0 };
+            eui64_t iid = { .uint64.u64 = 0, };
```

Comment:
We had some kind of problem with assignments like that and C++ (might be long gone). Just to be on the safe side: can you split this up into two lines:

```C
eui64_t iid;
iid.uint64.u64 = 0;
```